.github/workflows/build-sim-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Build & Push Sim Image
+on:
+  push:
+    branches:
+      - 'main'
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  IMAGE_NAME: sim-build
```

Comment:
drone-simulator-build?

Reply:
That variable isn't used. So I removed it.